src/feeds/InvPriceFeed.sol

```diff
@@ -39,9 +42,11 @@ contract InvPriceFeed {
     IChainlinkFeed public constant ethToUsd =
         IChainlinkFeed(0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419);
 
-    uint256 public constant ethK = 0;
+    uint256 public constant ethK = 1;
     uint256 public constant invK = 1;
 
+    uint256 public constant ethHeartbeat = 1 hours;
```

Comment:
We may want to be able to change heart beat time, or at least add a little time to it beyond the strict heartbeat. I'm not sure Chainlink has ever changed the heartbeat of an active oracle, but I believe they possess the ability to do so.